src/seeder/task-seeder.ts

```diff
@@ -0,0 +1,55 @@
+import { CreateTaskRequest } from './../types/dto/tasks.dto'
+import * as faker from '@faker-js/faker'
+
+import { WorkflowStateResponse } from './../types/dto/workflowStates.dto'
+
+const apiUrl = 'http://localhost:3000'
+
+const token = ''
```

Comment:
If the token is empty, won't this stop us from extracting the value of workspaceId, etc?